triggers/service/authentication.md

```diff
@@ -53,3 +53,60 @@ We can directly check that the credential is still valid before requesting a new
 For a party using the trigger service after the first time, we will look up their service account and see if there is a valid SA credential associated with it.
 Depending on that, we enter the flow at step 4 (requesting a new SA credential) or step 7 (`login` with an existing credential).
 
+## Example flow using the command line
+
+In one terminal window, run a sandbox which the auth service will use as its "admin ledger".
+(It is an implementation choice of the auth service to use a DAML ledger for managing users, service accounts, credentials, etc).
+```
+❯ daml sandbox
+```
+
+In another terminal window, build and run the auth service.
+We run with the "test mode" which allows us to authorize users using basic username/password credentials.
+```
+❯ make clean assembly
+❯ DABL_AUTHENTICATION_SERVICE_TEST_MODE=true java -jar target/scala-2.12/authentication-service-assembly-0.1.jar
+```
+
+In a third window, we'll execute requests to the auth service using `curl`.
+
+```
+❯ curl -X POST localhost:8089/sa/secure/authorize -u user:pass
+{"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6ImEtcy1lMDZmZDVjMy00NWM1LTRjZGQtYTI1OC1iMmIzMmIyNjA3NDYifQ.eyJpc3MiOiJsb2NhbGhvc3QvbG9naW4iLCJzdWIiOiJ1c2VyIiwiZXhwIjoxNTk0ODI3NTIwLCJsZWRnZXJJZCI6Ii5hdXRoIiwicGFydHkiOiJ1c2VyIiwicmlnaHRzIjpbInJlYWQiLCJ3cml0ZTpjcmVhdGUiLCJ3cml0ZTpleGVyY2lzZSJdfQ.NiOHuA3Zm6yAs2ThSlXzlGyffbjauJLBjlnD8k8Qi-BBsya45bD0k4cLr-NMcUeabyoEHWVdDuPm7CoUCRkMqePBU5Q9FcwNyU3pqUdiRwmsLLEnFfaSi5mXV9JDH8qlw59LBLXOu5_D2i3p5mC4WB6hvLjqocJG7hguG5pmYZ4"}
+```
+
+```
+❯ curl -X GET localhost:8089/sa/secure/me --oauth2-bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6ImEtcy1lMDZmZDVjMy00NWM1LTRjZGQtYTI1OC1iMmIzMmIyNjA3NDYifQ.eyJpc3MiOiJsb2NhbGhvc3QvbG9naW4iLCJzdWIiOiJ1c2VyIiwiZXhwIjoxNTk0ODI3NTIwLCJsZWRnZXJJZCI6Ii5hdXRoIiwicGFydHkiOiJ1c2VyIiwicmlnaHRzIjpbInJlYWQiLCJ3cml0ZTpjcmVhdGUiLCJ3cml0ZTpleGVyY2lzZSJdfQ.NiOHuA3Zm6yAs2ThSlXzlGyffbjauJLBjlnD8k8Qi-BBsya45bD0k4cLr-NMcUeabyoEHWVdDuPm7CoUCRkMqePBU5Q9FcwNyU3pqUdiRwmsLLEnFfaSi5mXV9JDH8qlw59LBLXOu5_D2i3p5mC4WB6hvLjqocJG7hguG5pmYZ4
+{
+  "user": "sa-user-21a4c943-57cc-46e7-b07e-380d2c780622"
+}
+```
+
+We'll write `--oauth2-bearer ...` in the following requests to refer to same bearer token as used above.
+Also note that the service account `request` below refers to a `test-ledger` id of the ledger we are
+granting authentication to (not the admin ledger).
+```
+❯ curl -X POST localhost:8089/sa/secure/request/test-ledger -H "Content-type: application/json" -d '{"nonce": "random-nonce"}' --oauth2-bearer ...
+
+❯ curl -X GET localhost:8089/sa/secure --oauth2-bearer ...
```

Comment:
Does this request do anything or is this just to view the output?

Reply:
Just views the output. This won't actually be used in the auth flow, though it's nice to confirm the previous command worked when running manually.